va/va_backend.h

```diff
@@ -485,8 +485,14 @@ struct VADriverVTable
             void               *descriptor      /* out */
         );
 
+        VAStatus (*vaSyncBuffer) (
+            VADriverContextP ctx,
+            VABufferID buf_id,
+            uint64_t timeout_ns
+        );
+
         /** \brief Reserved bytes for future use, must be zero */
-        unsigned long reserved[57];
+        unsigned long reserved[55];
```

Comment:
The type is unsigned long, so the size of elements in this array is 56 after adding a function callback.

Reply:
Thanks. Fixed.